core/control-plane/transfer-core/src/main/java/org/eclipse/dataspaceconnector/transfer/core/CoreTransferExtension.java

```diff
@@ -136,7 +139,7 @@ public void initialize(ServiceExtensionContext context) {
         var provisionManager = new ProvisionManagerImpl(monitor);
         context.registerService(ProvisionManager.class, provisionManager);
 
-        var waitStrategy = context.hasService(TransferWaitStrategy.class) ? context.getService(TransferWaitStrategy.class) : new ExponentialWaitStrategy(DEFAULT_ITERATION_WAIT);
+        var waitStrategy = context.hasService(TransferWaitStrategy.class) ? context.getService(TransferWaitStrategy.class) : new ExponentialWaitStrategy(context.getSetting(TRANSFER_STATE_MACHINE_ITERATION_WAIT_MILIS, 5000));
```

Comment:
As before, please use the constant instead of the hardcoded value and extract a variable